packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -36,6 +36,13 @@ const Container = styled.div`
     border-bottom: 1px solid ${({ theme }) => theme.STROKE_LIGHT_GREY};
     font-size: ${variables.FONT_SIZE.TINY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    ${({ isClickable, theme }) =>
+        isClickable &&
+        `
+        cursor: pointer;
+        &:hover {
+            background: ${theme.BG_WHITE_ALT_HOVER};
```

Comment:
nit: `BG_LIGHT_GREY` looks better imo :)

Reply:
I think it's very hard to see the background with `BG_LIGHT_GREY `, and it will be even worse on bad-quality monitors:

https://github.com/trezor/trezor-suite/assets/858321/6ada8fbe-c463-4f27-b1e2-26411925d0eb

Reply:
yeah, you are right, watching this from a bad quality monitor.... x)